project-management/1-1-meetings/Sophia-Batchelor/weekly-meetings-2024.md

```diff
@@ -0,0 +1,34 @@
+# Sophia and Bastian Weekly Meeting Notes 
+
+* [23 January 2024](#23-January-2024)
+* [15 January 2024]
+
+## 23 January 2024
+
+### Project Updates [[taskboard]](https://github.com/orgs/aim-rsf/projects/15/views/2)
+* **Inclusive Events Chapter / PPIE Chapter**
+  * Currently working on pulling out sections that are relevant to members of the public without disabilities and reworking sentences.
+  * ETA 2 weeks for working draft
+* **Objectives**
+  * Need updating but currently "in workflow task" so cannot be edited on my side 
```

Comment:
```suggestion
* **Objectives**
  * Needed updating but currently "in workflow task" so cannot be edited on my side 
  * BGT "sent back" the first draft in Cezanne so that they can be edited again!
```